packages/amazonq/src/extensionShared.ts

```diff
@@ -93,6 +95,34 @@ export async function activateShared(context: vscode.ExtensionContext) {
 
     // enable auto suggestions on activation
     await CodeSuggestionsState.instance.setSuggestionsEnabled(true)
+
+    await telemetry.auth_userState.run(async () => {
+        telemetry.record({ passive: true })
+
+        const firstUse = AuthUtils.ExtensionUse.instance.isFirstUse()
+        const wasUpdated = AuthUtils.ExtensionUse.instance.wasUpdated()
+
+        if (firstUse) {
+            telemetry.record({ source: ExtStartUpSource.FirstStartUp })
+        } else if (wasUpdated) {
+            telemetry.record({ source: ExtStartUpSource.Update })
+        } else {
+            telemetry.record({ source: ExtStartUpSource.Reload })
+        }
+
+        const authState = (await AuthUtil.instance.getChatAuthState()).codewhispererChat
```

Comment:
BiD and IdC will now be at parity in terms of the Q features they offer, so I think we will only need to use `amazonQ` instead of `codewhispererChat`. We only had this discrepancy before since only some of Q worked with BiD

Reply:
Good call out. I am going to add researching this as a follow up item to our checklist. Last time I tested this, Q was working fine despite returning false for .amazonq but not .codewhispererChat.